### ISSUE TITLE
Investigate other map pages

### ISSUE THREAD

**Issue by Nopey** [ryanmcnz/maplunky#5]
I've been exploring the spelunky binary, and found 
IDS_ARENA_X, IDS_ARENA_DLC_1, and IDS_ARENA_DLC_2
I'm working on getting the checksums and names for these pages of maps.

**Comment by LiamKarlMitchell** [comment]
Yeah, I loaded those up at some point by copying the map data to other maps.
Obviously without the DLC Textures or w/e.

Some were weird, I suppose it might be unfinished content?
A hint at a possible expansion, could be something to ask Derek Yu about?

[Edit] Or maybe I am just out of the loop I only have it on steam and have not checked if I can buy any DLC yet :).
http://www.mtv.com/news/2466514/new-spelunky-dlc-adds-new-characters-and-deathmatch-maps/

Others rumor they are unlocked by killing all bosses?
http://steamcommunity.com/app/239350/discussions/0/558751364363858979/

Could be console exclusive somehow and just "turned off" on PC version.

**Comment by ghost** [comment]
The DLC content is part of the PC version, it was sold like DLCs in XBLA but it comes with the game in Steam, it's the extra characters and the boss arenas.

**Comment by Nopey** [comment]
X, DLC_1, and DLC_2 seem to be OLD BITEY,YETI KING, ALIEN QUEEN. (in no particular order)
